keras_cv/layers/preprocessing/random_augmentation_pipeline.py

```diff
@@ -99,11 +105,15 @@ def _augment(self, inputs):
 
     def get_config(self):
         config = super().get_config()
+        if self._layers_empty:
```

Comment:
This looks like a hack to make unit tests work.

Does passing self.layers change any behavior other than making the UT fail?  If so, just update the unit test to take a single layer.

Reply:
@LukeWood 

No, it is so that we don't serialize the Lambda layer. The layer will always generate a new Lambda layer object when it gets layers=[]. 

It also does pass the UT, but that's not the main motive.